src/brasil/gov/portal/upgrades/v10700/handler.py

```diff
@@ -19,7 +21,7 @@
 
 def atualiza_produtos_terceiros(context):
     """ Atualiza os profiles de produtos de terceiros."""
-    profiles = (
+    profiles = [
```

Comment:
tuple and lists are both sequences; there's no need to change this if you are not modifying it later